### ISSUE TITLE
new orthologs to add

### ISSUE THREAD

**Issue by ValWood** [japonicusdb/japonicus-curation#26]
[main 790dc9e] more orthologs
SJAG_03640      SPBP19A11.02c
SJAG_01396      SPAPB2B4.06
SJAG_01849      SPBC651.12c

new orthologs added

**Comment by ValWood** [comment]
new orthologs added
SJAG_06135	SPAC1486.11
SJAG_03339	SPAC1565.03
SJAG_05607	SPAC4H3.16
SJAG_00556	SPBC2A9.14
SJAG_04861	SPBC660.09
SJAG_00422	SPAC29E6.09 
SJAG_00264	SPAPB1A10.13
SJAG_00789	SPCPB16A4.07
SJAG_06038	SPAC806.05
SJAG_05560	SPCC777.11
SJAG_06268	SPBC660.17c
SJAG_06109	SPAC17A5.19

**Comment by ValWood** [comment]
SJAG_06097	SPCC330.21 
SJAG_00191	SPAC57A10.06
SJAG_04776	SPAC1006.08
SJAG_04780	SPAC1250.02

**Comment by ValWood** [comment]
these are added last week too
SJAG_00908	SPBC31F10.08
SJAG_06392	SPAPB8E5.10
SJAG_03640	SPBP19A11.02c 
SJAG_01396	SPAPB2B4.06 
SJAG_01849	SPBC651.12c

**Comment by ValWood** [comment]
@snezhkaoliferenko   these are the new orthologs I added so far. I still have 120 to do. It is slow...

**Comment by snezhkaoliferenko** [comment]
Hey! @ValWood This is awesome, thank you! 
A couple questions:

1) do we want to keep potential pombe-specific annotations? E.g. SJAG_06392 comes up as 'conserved fungal protein, with meiosis specific splicing' - but do we know it's spliced in some specific way in japonicus?

2) SJAG_03339, SJAG_00556, this comes up as 'Schizosaccharomyces pombe specific protein' - should it be changed in both JaponicusDB and PomBase - to 'Schizosaccharomyces specific protein' or some such?

**Comment by ValWood** [comment]
Yes, Once done I will go through this list and fix the products in pombase to be more generic (and update the taxonomic distribution) . This will automatically fix in japonicus DB.

I will open a ticket on Pombase tracker for this now so I don't forget!

**Comment by ValWood** [comment]
SJAG_07002	SPCC1753.03c (rec)

**Comment by ValWood** [comment]
SJAG_07001 	SPBC21B10.15
SJAG_01515	SPAC5D6.08c
SJAG_06560	SPAC6B12.18
SJAG_02639	SPBC1703.09
SJAG_01479	SPAC12G12.09
SJAG_03571	SPBC1A4.11c

**Comment by ValWood** [comment]
Todays orthologs so far  ;(
SJAG_00046	SPAC144.19 ||  no PSI/jackhmmr/fasta/ prl46 syntenic orf identified by jackhammer from prl46 
SJAG_06270	SPAC2C4.10c || jackhmmr & syntenic

**Comment by snezhkaoliferenko** [comment]
oh cool a FAR component

**Comment by ValWood** [comment]
Yes, but I can't get this yet outside of schizosaccharomyces. Anyway, previously it was annotated pombe only - at least I can change that ;)
At the end   (well later someday), can try the searches  of the schizo specific with different parameters to see if they can be pushed further....

**Comment by ValWood** [comment]
SJAG_02976	SPAC25B8.20 || smp1 
SJAG_01691	SPCC584.16c || 
SJAG_04078	SPBC17D1.17  || tam11
SJAG_05269	SPAC8C9.09c  || mug129 
SJAG_03605	SPCC1235.12c  || mug146 
SJAG_04026	SPBC215.04  || git11 
SJAG_05307	SPAC22A12.02c  || mug103

**Comment by ValWood** [comment]
SJAG_05950	SPCC16C4.04 
[main 70ed4f6] new ortholog, end of round1

**Comment by ValWood** [comment]
[main ffb890c] spo15
SJAG_05309	SPAC1F3.06c

**Comment by ValWood** [comment]
SJAG_07007      SPCC1259.05
SJAG_07008      SPBC3B8.10
SJAG_07010      SPAC56F8.07
SJAG_07011      SPCC18.11
SJAG_07012      SPAC630.11
SJAG_07009      SPBC8D2.23

[main c146ada] new orthologs

**Comment by ValWood** [comment]
SJAG_07015      SPAC652.01
SJAG_07013      SPAC3G6.13c
SJAG_07014      SPAC3F10.18c
SJAG_07016      SPBC106.07c
SJAG_07017      SPBC26H8.13c

[main 5040220] orthologs for new genes -> 7016

**Comment by snezhkaoliferenko** [comment]
am low key admiring this @ValWood

**Comment by ValWood** [comment]
It's an old skill ;)  It seems like riding a bike though.....

I will do the low hanging fruit for the paper to demonstrate why genomes need tending, but I suspect  there will be many more. 

Some of them are annoying me  - I know they are there but I just can't get a handle on them. After round one I can look at alternative search strategies/parameters etc....but this may need to wait for a while

**Comment by ValWood** [comment]
SJAG_06638	SPMTR.01
SJAG_05269	SPAC8C9.09c
SJAG_01247	SPAC8C9.04
